**Tema seleccionado: Análisis Estadístico de Indicadores de Salud para la Predicción del Riesgo de Diabetes**

La diabetes mellitus constituye una de las principales causas de morbilidad y mortalidad a nivel mundial, con una prevalencia en constante aumento que la posiciona como un desafío crítico para los sistemas de salud pública. Su etiología multifactorial —que integra determinantes demográficos, estilos de vida, antecedentes familiares y marcadores clínicos— la convierte en un fenómeno complejo cuyo estudio demanda aproximaciones estadísticas rigurosas y multidimensionales. La identificación temprana de factores de riesgo y la caracterización de subpoblaciones vulnerables son esenciales para el diseño de intervenciones preventivas efectivas y la optimización de recursos sanitarios.

Este proyecto aborda la predicción y caracterización estadística del riesgo de diabetes mediante el estudio de registros sintéticos pero médicamente realistas, permitiendo aplicar técnicas estadísticas avanzadas del curso en un contexto de alta relevancia clínica y social.

**Preguntas de investigación:**

1. **(...)**
2. **(...)**
3. **(...)**

### Fuente de datos

Los datos fueron obtenidos del repositorio **Kaggle** en el dataset *"Diabetes Health Indicators Dataset"*:

**🔗 Enlace**: [https://www.kaggle.com/datasets/mohankrishnathalla/diabetes-health-indicators-dataset](https://www.kaggle.com/datasets/mohankrishnathalla/diabetes-health-indicators-dataset)

**📊 Características del dataset**:
- **Filas**: 100,000 registros (1 por paciente)
- **Columnas**: 35+ variables
- **Formato**: CSV limpio y preprocesado

### Categorías de variables disponibles

| Categoría | Variables principales | Ejemplo de rangos |
|-----------|----------------------|------------------|
| **Demográficas** (6) | `age`, `gender`, `ethnicity`, `education_level`, `income_level`, `employment_status` | 18-90 años, Male/Female/Other |
| **Estilo de vida** (7) | `smoking_status`, `alcohol_consumption_per_week`, `physical_activity_minutes_per_week`, `diet_score` | 0-30 bebidas/semana, 0-10 dieta |
| **Antecedentes médicos** (3) | `family_history_diabetes`, `hypertension_history`, `cardiovascular_history` | 0=No, 1=Yes |
| **Clínicas** (17) | `bmi`, `systolic_bp`, `glucose_fasting`, `hba1c`, `cholesterol_total` | BMI: 15-45, HbA1c: 4-14% |

**Variable objetivo principal**: `diagnosed_diabetes` (0=No, 1=Yes)

Los datos sintéticos preservan la privacidad mientras mantienen validez analítica, siendo ideales para los requisitos del proyecto académico.


### 3.2. Análisis Exploratorio de Datos (EDA)

Como primer paso del análisis exploratorio, se calcularon estadísticos descriptivos básicos (medidas de tendencia central y dispersión) para algunas de las variables numéricas más relevantes del conjunto de datos. 

Estos resúmenes permiten caracterizar el perfil general de la población estudiada, identificar valores típicos y rangos de variación, y detectar posibles patrones iniciales relacionados con el riesgo de diabetes antes de aplicar técnicas estadísticas más avanzadas.


In [17]:
import pandas as pd

# Cargar el dataset
df = pd.read_csv('Diabetes_and_LifeStyle_Dataset .csv')

# Variables asociadas al estilo de vida
lifestyle_vars = [
    "alcohol_consumption_per_week",
    "physical_activity_minutes_per_week",
    "diet_score",
    "screen_time_hours_per_day",
    "sleep_hours_per_day"
]

clinical_vars = [
    "bmi",
    "insulin_level",
    "triglycerides",
    "glucose_fasting",
    "hba1c",
    "systolic_bp",
    "diastolic_bp",
    "heart_rate",
    "cholesterol_total"
]

demographic_vars = [
    "Age"]


summary = df[clinical_vars + lifestyle_vars + demographic_vars].select_dtypes(include="number").agg([
    'count',
    'mean',
    'median',
    'std',
    'min',
    lambda x: x.quantile(0.25),
    lambda x: x.quantile(0.50),
    lambda x: x.quantile(0.75),
    'max'
]).T

summary.columns = [
    'count', 'mean', 'median', 'std',
    'min', '25%', '50%', '75%', 'max'
]

summary


count        mean  median        std  \
bmi                                 97297.0   25.615093   25.60   3.588387   
insulin_level                       97297.0    9.063591    8.79   4.956342   
triglycerides                       97297.0  121.466407  121.00  43.393422   
glucose_fasting                     97297.0  111.121792  111.00  13.590382   
hba1c                               97297.0    6.520338    6.52   0.813710   
systolic_bp                         97297.0  115.800127  116.00  14.287977   
diastolic_bp                        97297.0   75.233964   75.00   8.206551   
heart_rate                          97297.0   69.628807   70.00   8.373731   
cholesterol_total                   97297.0  185.984234  186.00  32.013395   
alcohol_consumption_per_week        97297.0    2.004543    2.00   1.418060   
physical_activity_minutes_per_week  97297.0  118.942886  100.00  84.441920   
diet_score                          97297.0    5.995214    6.00   1.780096   
screen_time_hours_per_day           97297.0    5.997031    6.00   2.469805   
sleep_hours_per_day                 97297.0    6.997708    7.00   1.093726   
Age                                 97297.0   50.192699   51.00  15.492557   

                                      min     25%     50%     75%     max  
bmi                                  15.0   23.20   25.60   28.00   39.20  
insulin_level                         2.0    5.09    8.79   12.45   32.22  
triglycerides                        30.0   91.00  121.00  151.00  344.00  
glucose_fasting                      60.0  102.00  111.00  120.00  172.00  
hba1c                                 4.0    5.97    6.52    7.07    9.80  
systolic_bp                          90.0  106.00  116.00  125.00  179.00  
diastolic_bp                         50.0   70.00   75.00   81.00  110.00  
heart_rate                           40.0   64.00   70.00   75.00  105.00  
cholesterol_total                   100.0  164.00  186.00  208.00  318.00  
alcohol_consumption_per_week          0.0    1.00    2.00    3.00   10.00  
physical_activity_minutes_per_week    0.0   57.00  100.00  160.00  833.00  
diet_score                            0.0    4.80    6.00    7.20   10.00  
screen_time_hours_per_day             0.5    4.30    6.00    7.70   16.80  
sleep_hours_per_day                   3.0    6.30    7.00    7.70   10.00  
Age                                  19.0   38.00   51.00   61.00   90.00

El análisis descriptivo se basa en una muestra de 97,297 individuos adultos, con una edad promedio de 50.2 años (rango: 19-90 años). El perfil clínico de la población se caracteriza por valores promedio que se sitúan en rangos elevados para varios factores de riesgo metabólico y cardiovascular. El índice de masa corporal (BMI) promedio es de 25.6 kg/m², ubicándose en la categoría de sobrepeso, con una distribución que abarca desde bajo peso (15.0 kg/m²) hasta obesidad grado II (39.2 kg/m²). Los biomarcadores glucémicos revelan un estado metabólico comprometido: la glucosa en ayunas presenta una media de 111.1 mg/dL y la hemoglobina glicosilada (HbA1c) de 6.52%, ambos valores en el umbral de diagnóstico para diabetes mellitus según criterios internacionales. El perfil lipídico muestra niveles promedio de triglicéridos de 121.5 mg/dL y colesterol total de 186.0 mg/dL, considerados en el límite alto de la normalidad.

Los parámetros cardiovasculares indican una presión arterial media de 115.8/75.2 mmHg y una frecuencia cardíaca en reposo de 69.6 lpm, valores generalmente dentro de rangos normales. Sin embargo, se observa una dispersión significativa, con casos de hipertensión sistólica (máximo 179 mmHg) e hipertensión diastólica (máximo 110 mmHg). El nivel de insulina promedio es de 9.06 μU/mL, con una desviación estándar considerable (4.96), lo que sugiere una amplia variabilidad en la resistencia a la insulina entre los individuos.

Respecto a los hábitos de estilo de vida, se identifica un patrón de actividad física insuficiente, con un promedio de 118.9 minutos semanales, por debajo de las recomendaciones de 150 minutos. El tiempo de pantalla diario es elevado (6.0 horas en promedio), mientras que la duración del sueño se aproxima a lo recomendado (7.0 horas). El consumo de alcohol es moderado (2.0 bebidas/semana) y la calidad de la dieta, evaluada mediante una puntuación de 0 a 10, es media (6.0). La distribución de estos factores muestra una notable heterogeneidad, con extremos que van desde el sedentarismo absoluto (0 minutos de actividad física) hasta una actividad vigorosa (833 minutos/semana), y desde una dieta muy pobre (puntuación 0) hasta una excelente (puntuación 10).

En conjunto, los datos describen una población con una carga significativa de factores de riesgo metabólico, particularmente relacionados con la regulación de la glucosa y el peso corporal. La coexistencia de estos marcadores clínicos con estilos de vida subóptimos, especialmente en actividad física y tiempo de pantalla, refuerza la noción de un perfil de riesgo elevado.